src/plugins/intel_gpu/include/intel_gpu/runtime/debug_configuration.hpp

```diff
@@ -3,7 +3,7 @@
 //
 
 #pragma once
-#include <cstring>
+#include <string>
```

Comment:
`<string>` doesn't work with clang
Maybe `<string.h>` ?

Reply:
as I can see we need `<string>` for `std::string`.
what API of `<string.h>` / `<cstring>` is used in this file?

Reply:
`strrchr`

Reply:
kept both includes `<string>` and `<cstring>`